--- networkmanager/PKGBUILD	2020-05-08 23:58:25.000000000 +0200
+++ PKGBUILD	2020-05-10 20:28:34.875697120 +0200
@@ -9,1 +9,1 @@
-pkgname=(networkmanager libnm nm-cloud-setup)
+pkgname=(networkmanager_auto libnm nm-cloud-setup)
@@ -30,6 +30,17 @@
 
 prepare() {
   cd NetworkManager
+
+  # NetworkManager jest uruchamiany przez systemd jeszcze przed uruchomieniem
+  # i odblokowaniem gnome-keyring'a. NetworkManager wyłącza dane połączenie
+  # z autoconnect'a jeśli nie jest w stanie pobrać sekretów z keyringa. Przez to
+  # po zalogowaniu, włączeniu i odblokowaniu keyringa, NetworkManager nie będzie
+  # podejmował kolejnych prób połączenia z internetem. Poniższy patch zmienia to
+  # zachowanie - NetworkManager po tej zmianie nadal będzie próbował uaktywnić
+  # dane połączenie nawet jeśli keyring jeszcze nie działa, pod warunkiem odpowiedniej
+  # konfiguracji połączenia (autoconnect-retries=0 w sekcji connection w pliku
+  # .nmconnection)
+  sed -i '0,/blocked = TRUE;/s//blocked = FALSE;/' src/nm-policy.c
 }
 
 build() {
@@ -87,7 +98,7 @@
   done
 }
 
-package_networkmanager() {
+package_networkmanager_auto() {
   depends=(libnm iproute2 polkit wpa_supplicant libmm-glib libnewt libndp libteam curl
            bluez-libs libpsl audit mobile-broadband-provider-info)
   optdepends=('dnsmasq: connection sharing'
@@ -99,6 +110,8 @@
               'openresolv: alternative resolv.conf manager')
   backup=(etc/NetworkManager/NetworkManager.conf)
   groups=(gnome)
+  provides=('networkmanager')
+  conflicts=('networkmanager')
 
   DESTDIR="$pkgdir" meson install -C build
 
