--- PKGBUILD	2020-08-11 20:28:47.000000000 +0200
+++ PKGBUILD.new	2020-08-11 21:11:38.748516911 +0200
@@ -1,6 +1,6 @@
 # Maintainer: Thorsten Töpper <atsutane-tu@freethoughts.de>
 
-pkgname=i3-wm
+pkgname=i3-wm-non-primary
 pkgver=4.18.2
 pkgrel=2
 pkgdesc="An improved dynamic tiling window manager"
@@ -38,6 +38,8 @@
   # https://github.com/i3/i3/issues/4159
   patch -Np1 -i ../0002-kick-tray-clients-before-destroying-the-bar.patch
 
+  patch -p0 -i ../outputs.patch
+
   autoreconf -fvi
 }
 
@@ -51,7 +53,12 @@
   cd build
   make DESTDIR="$pkgdir" install
   install -Dt "$pkgdir/usr/share/man/man1" -m644 man/*.1
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 ../i3-$pkgver/LICENSE
+  install -Dt "$pkgdir/usr/share/licenses/i3-wm" -m644 ../i3-$pkgver/LICENSE
 }
 
+source+=(outputs.patch)
+sha256sums+=('534a70010a2d1f12724144055d13e71540f98c91346e315add1e64cdba4ffad3')
+provides=('i3-wm')
+conflicts=('i3-wm')
+
 # vim:set ts=2 sw=2 et:
